pydantic/analyzed_type.py

```diff
@@ -0,0 +1,241 @@
+from __future__ import annotations
+
+import sys
+from typing import TYPE_CHECKING, Any, Dict, Generic, Iterable, Set, TypeVar, Union, overload
+
+from pydantic_core import CoreConfig, CoreSchema, SchemaSerializer, SchemaValidator, core_schema
+from typing_extensions import Literal
+
+from pydantic.config import ConfigDict
+from pydantic.json_schema import DEFAULT_REF_TEMPLATE, GenerateJsonSchema
+
+from ._internal import (
+    _generate_schema,
+    _typing_extra,
+)
```

Comment:
```suggestion
from ._internal import _generate_schema, _typing_extra
```
ruff can be bad at correcting imports layout.